### PR TITLE
Feat/landing

### PR DESCRIPTION
**Pull request by bhavyaj12**
### Added Home Page complete

- Installed bootstrap, react-bootstrap, react-router-bootstrap, mui icons
- Added nav top, footer, navside, add post, post card, suggestions list (components)
- Added router and basic routes
- Added landing page, home feed page (pages)
- Added mockman

**Please check app is RESPONSIVE**

### What not to check
- Navlinks
- Any other pages and routes
- Any functionality

image
image

### CODE REVIEW

File: src/assets/icons-logos/icons.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import AddPhotoAlternateIcon from "@mui/icons-material/AddPhotoAlternate";
```
Comment (chanchal16) [inline src/assets/icons-logos/icons.js:1]: Good approach of creating a separate file to import your assets.

File: src/components/SuggestionList.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import { Image } from "react-bootstrap";
+
+const SuggestionList = () => {
+  return (
+    <div className="suggestions-wrapper bg-light py-2">
+      <h5 className="mt-4">Suggestions for you</h5>
+      <div className="flex-row-centre mt-3">
```
Comment (chanchal16) [inline src/components/SuggestionList.jsx:7]: Here you can create users data and map through it.

File: src/routes/Routes.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { Routes, Route } from "react-router-dom";
+import Mockman from "mockman-js";
+import {
+  HomePage,
+  UserHome
+} from "pages";
+// import PrivateRoutes from "./PrivateRoutes";
```
Comment (chanchal16) [inline src/routes/Routes.jsx:7]: Remove the commented code.

**Review by chanchal16 — APPROVED** [review]
Code looks clean. Left one suggestion else good to go ahead.

File: src/App.css [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+.nav-link.link-dark:hover {
+  background-color: var(--social-secondary);
+}
+p.lead {
+  font-weight: 500;
+}
+
+@media screen and (max-width: 1000px) {
+  #nav-side {
+    display: none !important;
```
Comment (manish591) [inline src/App.css:29]: Try to avoid important

**Review by manish591 — APPROVED** [review]
Your code is looking good. Properly indented and readable as well. 

In your `app.js` file, you are creating components using the `function declaration syntax`  while in the rest of the components you have used the `arrow function syntax`. So, you can change this to maintain consistency.

**Pull request merged**